packages/synthetix-main/contracts/modules/core/CollateralModule.sol

```diff
@@ -35,12 +36,15 @@ contract CollateralModule is ICollateralModule {
 
     error InsufficientAccountCollateral(uint amount);
 
+    // set address oracle-managerr
+    //  set nodeId snx
+
     /**
      * @dev See {ICollateralModule-configureCollateral}.
      */
     function configureCollateral(
         address collateralType,
-        address priceFeed,
```

Comment:
please note that this function arguments has been modified in another PR https://github.com/Synthetixio/synthetix-v3/pull/1175 . should be fine whoever merges first, but just wanted to lyk